python/tests/test_io.py

```diff
@@ -25,11 +25,12 @@
 from rikai.conf import get_option, CONF_RIKAI_IO_HTTP_AGENT
 
 WIKIPEDIA = (
-    "https://upload.wikimedia.org/wikipedia/commons/thumb/a/ad/"
-    "Commodore_Grace_M._Hopper%2C_USN_%28covered%29.jpg/819px-Commodore_"
-    "Grace_M._Hopper%2C_USN_%28covered%29.jpg"
+    "https://upload.wikimedia.org/wikipedia/commons/a/ad/"
+    "Commodore_Grace_M._Hopper%2C_USN_%28covered%29.jpg"
 )
 
+USER_AGENT = "User-Agent: Rikai/0.0.10 (rikai-dev@eto.ai)"
```

Comment:
should we move this to `io.py`?

Reply:
This is just used here to check the result coming from io.py